plugins/woocommerce/tests/e2e-pw/tests/admin-tasks/payment.spec.js

```diff
@@ -52,18 +52,18 @@ test.describe( 'Payment setup task', () => {
 			.catch( () => {} );
 
 		// fill in bank transfer form
-		await page.fill( `//input[@placeholder=${getTextForLanguage()['AccountnameInQuotes']}]`, 'Savings' );
-		await page.fill( `//input[@placeholder=${getTextForLanguage()['AccountnumberInQuotes']}]`, '1234' );
-		await page.fill( `//input[@placeholder=${getTextForLanguage()['BanknameInQuotes']}]`, 'Test Bank' );
-		await page.fill( `//input[@placeholder=${getTextForLanguage()['SortcodeInQuotes']}]`, '12' );
-		await page.fill( `//input[@placeholder=${getTextForLanguage()['IBANInQuotes']}]`, '12 3456 7890' );
-		await page.fill( `//input[@placeholder=${getTextForLanguage()['BICSwiftInQuotes']}]`, 'ABBA' );
-		await page.click( `text=${getTextForLanguage()['Save']}` );
+		await page.fill( `//input[@placeholder=${getTranslationFor('"Account name"')}]`, 'Savings' );
```

Comment:
Why do we need to define the string between double quotes here?

Reply:
Great question @alopezari !

The reason we have to do this is that the original locators were wrapping up some of the references such as placeholders in double quotes, which were, in turn, inside single quotes as can be seen below:

```
		await page
			.locator( '//input[@placeholder="Account name"]' )
			.fill( 'Savings' );
```

If they were not wrapped in double quotes then they would not function inside the string which is bookended with single quotes. Hopefully this makes sense!